app/models/item.rb

```diff
@@ -1,5 +1,16 @@
-class Item < ApplicationRecord 
-  belongs_to :merchant 
-  has_many :invoice_items 
-  has_many :invoices, through: :invoice_items 
-end 
\ No newline at end of file
+class Item < ApplicationRecord
+  belongs_to :merchant
+  has_many :invoice_items
+  has_many :invoices, through: :invoice_items
+
+  def date_with_most_sales
+    invoices
+    .joins(:invoice_items)
+    .select("invoices.created_at, invoice_items.quantity as item_quantity")
+    .order(item_quantity: :desc)
+    .order('invoices.created_at desc')
```

Comment:
Oh wow! I didn't know you could use the order function twice. So this orders everything by it's quantity, and then if the quantity is the same it will then order it by the most recent created? Is that correct?

This is is a great way to handle that edge case. I made a comment in the item_spec file about maybe writing a test that checks for that explicitly.

Reply:
Awesome you found a way to use `order` twice! That definitely helps with the issue we were having.

Reply:
Now we check for it explicitly :)